src/main/webapp/app/exercises/shared/exercise/exercise.utils.ts

```diff
@@ -178,23 +178,19 @@ export const isStartPracticeAvailable = (exercise: Exercise, participation?: Stu
 
 /**
  * Checks whether the given exercise is eligible for receiving manual results.
- * This is the case if the user is at least a tutor and the exercise itself is a programming
- * exercise for which manual reviews have been enabled. The due date also has to be in the past.
+ * If it is a programming exercise, the due date also has to be in the past.
  *
  * @param exercise
  */
-export const areManualResultsAllowed = (exercise: Exercise) => {
-    if (exercise.type !== ExerciseType.PROGRAMMING) {
+export const areManualResultsAllowed = (exercise: Exercise): boolean => {
+    if (exercise.type === ExerciseType.QUIZ) {
         return false;
+    } else if (exercise.type !== ExerciseType.PROGRAMMING) {
```

Comment:
redundant else after return; The following else as well

Reply:
It's redundant but makes it even clearer that only one option is possible, makes sure that changing the return to something else will still only go into one case and also produces shorter code